.github/workflows/ci.yml

```diff
@@ -0,0 +1,123 @@
+name: CI
+on:
+  # Run on all pushes to master (including merge of PR)
+  push:
+    branches: master
+  # Run on all changes in pull requests
+  pull_request:
+  # Run CI once per day (at 06:00 UTC)
+  # schedule:
+  #   - cron: '0 6 * * *'
+
+jobs:
+  nikola:
+    name: Basic tests (Python ${{ matrix.python }} on ${{ matrix.image }})
+    strategy:
+      fail-fast: false
+      matrix:
+        python:
+          - 3.5
+          - 3.6
+          - 3.7
+          - 3.8
+        image:
+          - ubuntu-18.04
+          - ubuntu-16.04
+        exclude:
+          - python: 3.5
+            image: ubuntu-18.04
+          - python: 3.6
+            image: ubuntu-16.04
+          - python: 3.7
+            image: ubuntu-16.04
+          - python: 3.8
+            image: ubuntu-16.04
+    runs-on: '${{ matrix.image }}'
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '${{ matrix.python }}'
+      - name: Install test requirements
```

Comment:
Before installing requirements, we need to run this:

    pip install -U --upgrade-strategy=eager pip setuptools wheel